codes/quantum/qubits/quantum_repetition.yml

```diff
@@ -28,7 +28,7 @@ protection: 'Bit-flip code detects bit-flip errors \(X\) on \(\left\lfloor (n-1)
 
 realizations:
   - 'NMR: 3-qubit phase-flip code \cite{arxiv:quant-ph/9802018,arXiv:1108.4842}, with up to two rounds of error correction in liquid-state NMR \cite{arXiv:1109.4821}.'
-  - 'Superconducting circuits: 3-qubit phase-flip and bit-flip code by Schoelkopf group \cite{arXiv:1109.4948}, IBM 15-qubit device \cite{arXiv:1709.00990}, Google experiment performing up to 8 error-correction cycles on 5 and 9 qubits \cite{arXiv:1411.7403}, and Google Quantum AI Sycamore utilizing from 5 to 21 qubits \cite{arXiv:2102.06132}. Continuous error correction protocols have been implemented on a 3-qubit superconducting circuit device \cite{arxiv:2107.11398}.'
+  - 'Superconducting circuits: 3-qubit phase-flip and bit-flip code by Schoelkopf group \cite{arXiv:1109.4948}, another 3-qubit bit-flip code \cite{arXiv:1411.5542}, 3-qubit phase-flip code up to 3 cycles of error correction \cite{arXiv:1508.01388}, IBM 15-qubit device \cite{arXiv:1709.00990}, IBM 43-qubit code on Rochester device \cite{arXiv:2004.11037}, Google Quantum AI Sycamore performing up to 8 error-correction cycles on 5 and 9 qubits \cite{arXiv:2102.06132}, Google Quantum AI Sycamore utilizing from 5 to 21 qubits \cite{arXiv:2102.06132}. Continuous error correction protocols have been implemented on a 3-qubit superconducting circuit device \cite{arxiv:2107.11398}.'
```

Comment:
"IBM Rochester device"
Also, you've deleted 1411.7403, which is a separate ref from 2102.06132, which does much larger codes than 5 or 9 physical qubits. Please fix.

Reply:
Fixed this in the new commit, sorry about that.